Makefile

```diff
@@ -270,11 +272,32 @@ debug-t4c/client/backup: LOG_LEVEL=DEBUG
 debug-t4c/client/backup: DOCKER_RUN_FLAGS=-it --entrypoint="bash" -v $$(pwd)/backups/backup.py:/opt/capella/backup.py
 debug-t4c/client/backup: run-t4c/client/backup
 
+tests/local-git-server: SHELL=./capella_loop.sh
+tests/local-git-server:
+	if [ "$(RUN_TESTS_AGAINST_T4C_SERVER)" == "1" ]; \
+	then \
+		docker build $(DOCKER_BUILD_FLAGS) -t $(DOCKER_PREFIX)$@:$$DOCKER_TAG --build-arg CAPELLA_VERSION=$$CAPELLA_VERSION tests/local-git-server; \
+		$(MAKE) PUSH_IMAGES=$(PUSH_IMAGES) IMAGENAME=$@ .push; \
+	fi
+
+tests/t4c-server-docker-images: SHELL=./capella_loop.sh
+tests/t4c-server-docker-images:
+	if [ "$(RUN_TESTS_AGAINST_T4C_SERVER)" == "1" ]; \
+	then \
+		$(MAKE) -C tests/t4c-server-docker-images PUSH_IMAGES=$(PUSH_IMAGES) CAPELLA_VERSION=$$CAPELLA_VERSION t4c/server/server; \
+	fi
+
 test: SHELL=./capella_loop.sh
-test: capella/readonly t4c/client/remote
+test: capella/readonly t4c/client/remote t4c/client/backup tests/local-git-server tests/t4c-server-docker-images
+	export CAPELLA_VERSION=$$CAPELLA_VERSION
 	source .venv/bin/activate
 	cd tests
-	pytest -o log_cli=true -s
+	if [ "$(RUN_TESTS_AGAINST_T4C_SERVER)" == "1" ]
+	then
+		pytest -o log_cli=true -s
+	else
+		pytest -o log_cli=true -s -m "not (t4c_server or t4c)"
+	fi
 
```

Comment:
This variant does not work in all cases. For example, if both are set to 1, the mark would be `t4c_server and t4c`, which is false for all tests (so no test is executed). I now just do it the other way around, i.e. the default condition is `not (t4c or t4c_server)`, which selects all tests without a marker, and in case of a 1 I add `or t4c/t4c_server` to it.